kserve-modelmesh-serving.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: kserve-modelmesh-serving
+  version: 0.12.0
+  epoch: 0
+  description: ModelMesh Serving is the Controller for managing ModelMesh, a general-purpose model serving management/routing layer.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kserve/modelmesh-serving.git
+      tag: v${{package.version}}
+      expected-commit: d0e56fa4ac7547e644c87d63dc97a14f612c0391
+
+  - uses: go/build
+    with:
+      packages: .
+      output: manager
+
+  - uses: strip
```

Comment:
I think this one also not needed, go/build uses a "-w" flag to strip the binary, afaik.